packages/enzyme-test-suite/test/ReactWrapper-spec.jsx

```diff
@@ -2965,6 +2965,64 @@ describeWithDOM('mount', () => {
 
       expect(mount(<Comp />).debug()).to.equal('<Comp />');
     });
+
+    describe('child components', () => {
+      class Child extends React.Component {
+        constructor(...args) {
+          super(...args);
+          this.state = { a: 'a' };
+        }
+
+        render() {
+          const { _ } = this.props;
+          const { a } = this.state;
+          return (
+            <div>
+              {_}
```

Comment:
This might also be easier to understand as `{_} - {a}`, `expect(wrapper.text().trim()).to.eql('1a');` wasn't super obvious what was going on